includes/admin/reporting/class-categories-reports-table.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Gateways Reports Table Class
+ *
+ * @package     EDD
+ * @subpackage  Admin/Reports
+ * @copyright   Copyright (c) 2015, Pippin Williamson
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       2.4
+ */
+
+// Exit if accessed directly
+if ( ! defined( 'ABSPATH' ) ) exit;
+
+// Load WP_List_Table if not loaded
+if ( ! class_exists( 'WP_List_Table' ) ) {
+	require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+}
+
+/**
+ * EDD_Categories_Reports_Table Class
+ *
+ * Renders the Download Reports table
+ *
+ * @since 2.4
+ */
+class EDD_Categories_Reports_Table extends WP_List_Table {
+
+	/**
+	 * @var int Number of items per page
+	 * @since 1.5
+	 */
+	public $per_page = 30;
+
+
+	/**
+	 * Get things started
+	 *
+	 * @since 1.5
+	 * @see WP_List_Table::__construct()
+	 */
+	public function __construct() {
+		global $status, $page;
+
+		// Set parent defaults
+		parent::__construct( array(
+			'singular'  => edd_get_label_singular(),    // Singular name of the listed records
```

Comment:
Not following, this is how we define most of these in any other list table for downloads.